packages/core-data/src/resolvers.js

```diff
@@ -292,7 +292,7 @@ export const canUser = ( action, resource, id ) => async ( { dispatch } ) => {
 		return;
 	}
 
-	const allowHeader = response.headers.get( 'allow' );
+	const allowHeader = response.headers?.get( 'allow' );
```

Comment:
Let's add a comment explaining that the `?.` operator needs to be used because the React Native `apiFetch` doesn't return the expected result.